CONTRIBUTING.rst

```diff
@@ -167,3 +167,30 @@ To speed up the integration process, it helps if on GitHub you `allow
 maintainers to edit your pull request
 <https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/>`_
 so they can fix small issues autonomously.
+
+.. rubric: Review guidelines
+
+A pull request will probably be reviewed by someone in a different location,
+maybe even a different time zone. Therefore, avoiding blockings and
+misunderstandings is key. The following ideas could help ease and speed
+up the review process:
+
+- Be clear when something is ready to reviewed (see `WIP` above).
+- If a PR requires changes, say so explicitly, but politely (use emojis!).
+- When something is ready to be merge say it explicitly. If the reviewer does
+not have permission to merge, it should ping the corresponding person.
+- When reviewing, rate your comments. This would help the other person know If
+a comment must be addressed (e.g. it has substantial implications) or it is
+something the reviewer thought of but is a matter of "take it or leave it". A
+proposed scale is:
+  1. Comment/Shelved: Something worth mentioning, a doubt or question (comment).
+  If it will be treated later on as part of a different issue it is "shelved".
```

Comment:
```suggestion
  If it will be treated later or as part of a different issue, it is "shelved".
```

Reply:
I actually meant:

- It will be treated later
- Or as part of a different issue.

But is not a good practice to do things without an issue. So the correction applies :) hehe